web/server/system/shell.go

```diff
@@ -47,15 +48,15 @@ func (self *Shell) GoTest(directory, packageName string, arguments []string) (ou
 // Functional Core:////////////////////////////////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////////
 
-func findGoConvey(directory, gobin, packageName string) Command {
-	return NewCommand(directory, gobin, "list", "-f", "'{{.TestImports}}'", packageName)
+func findGoConvey(directory, gobin, packageName, tagsArg string) Command {
+	return NewCommand(directory, gobin, "list", "-f", "'{{.TestImports}}'", tagsArg, packageName)
```

Comment:
all of these auxilliary commands actually needed to take -tags too. Otherwise they simply wouldn't find any files, for example :(